rest-client/src/main/java/org/thingsboard/rest/client/RestClient.java

```diff
@@ -1356,13 +1356,14 @@ public PageData<Device> getTenantDevices(String type, PageLink pageLink) {
                 }, params).getBody();
     }
 
-    public PageData<DeviceInfo> getTenantDeviceInfos(String type, DeviceProfileId deviceProfileId, PageLink pageLink) {
+    public PageData<DeviceInfo> getTenantDeviceInfos(String type, Boolean active, DeviceProfileId deviceProfileId, PageLink pageLink) {
         Map<String, String> params = new HashMap<>();
         params.put("type", type);
+        params.put("active", active != null ? active.toString() : null);
```

Comment:
When `active` request param is absent, API should return all devices (active and inactive). So, please add this param only if supplied `active` is not null.

Reply:
yes, Because the project needs to use rest-client sdk to search for devices through active, I found that getTenantDeviceInfos does not support active search, so I modified the rest-client sdk of ThingsBoard, then mvn install, and then introduced rest-client.jar in my project. When active=null is tested in the project, both activated and inactive devices are returned.
example:
![image](https://github.com/user-attachments/assets/ca8ff788-6f2d-4e8c-98c2-1d6c7f06f167)
![image](https://github.com/user-attachments/assets/d13a5748-6471-4f2c-a29f-50fead68add9)